david_goodedit/src/components/Tutorial.jsx

```diff
@@ -9,8 +9,8 @@ import QuoteTuto from './QuoteTuto'
 
 function Tutorial() {
   return (
-    <div>
-      <h1 className="titleSection">Tutoriel</h1>
+    <div className=':root'>
```

Comment:
:root is not a class name: it is applied to root element of the document... and inherited by every elements, so no need to put it here :)